demo-shell/src/app/components/process-service/process-service.component.ts

```diff
@@ -213,6 +216,9 @@ export class ProcessServiceComponent implements AfterViewInit, OnDestroy, OnInit
         }
         this.sub = this.route.params.subscribe(params => {
             const applicationId = params['appId'];
+
+            this.filterObj = params['filterId'] ? { id: params['filterId'] } : { name: 'MY tasks' };
```

Comment:
We should never use the string, this work only when the language is english but if the language is a different one the name will be different. Could you use the index 0 ?

Reply:
Done.